.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,19 @@
+## What this PR includes?
 <!-- 
-Please refer to the issue or JIRA ticket to close like `Closes #14` or 
-`Fixes [CLIENT-1]`
+Explain what this PR implements:
+In case you are fixing any specific issue, please refer to it with `Fixes #issue_number`.
+In case you are implementing a new feature, please write a brief description about it.
+As an optional step, you can link or add any useful external documentation to give more context about the proposed changes (for example: design/architecture documents, figma links, screenshots, etc.).
 -->
-
-# What? <!-- what is this PR? -->
 ...
 
-# Why? <!-- Explain the reason -->
-...
+## How to test it?
+<!-- 
+Explain how to test the feature (or fix) for someone who doesn't know anything about this implementation:
+At very least add the specific URL from which to test the build and add to it any param you think it would be needed.
```

Comment:
This should be done by the CI automatically, anyway I agree that if any param is needed, that automatic comment with the URL should be edited to reflect it.